data/strings/strings.txt

```diff
@@ -5149,8 +5149,53 @@
     zh-Hant = 在螢幕上顯示
 
   [pref_map_style_title]
+    comment = Settings «Map» category: «Night style» title
```

Comment:
These are old, restored translations, right?

Reply:
Yes.
This string can be removed in future when we enable themes sync and move this row in General settings section on the Android